frontend/src/components/static_pages/team.jsx

```diff
@@ -1,5 +1,15 @@
 import React, { Component } from "react";
 import { Container, Col, Row } from "react-bootstrap";
+import "./team.css";
+import Picture1 from "../../assets/action-shots/3 rows of donations lined up.jpg";
```

Comment:
its usually better to store test under the public folder so we could reference them easier but this is solid 👍